### ISSUE TITLE
KeyError: 'getpwuid(): uid not found' if userid does not exist on system

### ISSUE THREAD

**Issue by pzelnip** [dhruvkb/pls#11]
## Description
On my system I have a number of folders that are owned by a user that does not exist on the system (502 in this case, this is `ls -Fla`  output):

```
drwxr-xr-x  18 adamparkin  staff       576 20 Mar  2021 site2/
drwxr-xr-x  19 adamparkin  staff       608 27 Apr  2021 skeleton/
drwxr-xr-x   6 502         staff       192  7 Dec  2014 slack-cli/
drwxr-xr-x   6 502         staff       192 14 Feb  2016 slackcmd/
```

Can see that two of those directories are owned by me (`adamparkin`) and two by a non-existent user (`502`).  When I run `pls -d` in this directory it crashes with:

```
 λ pls -d
Traceback (most recent call last):
  File "/Users/adamparkin/.local/bin/pls", line 8, in <module>
    sys.exit(main())
  File "/Users/adamparkin/.local/pipx/venvs/pls/lib/python3.9/site-packages/pls/main.py", line 34, in main
    write_output(node_list)
  File "/Users/adamparkin/.local/pipx/venvs/pls/lib/python3.9/site-packages/pls/output/table.py", line 114, in write_output
    tabulate_node(table, node)
  File "/Users/adamparkin/.local/pipx/venvs/pls/lib/python3.9/site-packages/pls/output/table.py", line 91, in tabulate_node
    data = node.table_row
  File "/opt/homebrew/Cellar/python@3.9/3.9.2_4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/functools.py", line 969, in __get__
    val = self.func(instance)
  File "/Users/adamparkin/.local/pipx/venvs/pls/lib/python3.9/site-packages/pls/models/node.py", line 185, in table_row
    cells.update(self.stat_cells)
  File "/opt/homebrew/Cellar/python@3.9/3.9.2_4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/functools.py", line 969, in __get__
    val = self.func(instance)
  File "/Users/adamparkin/.local/pipx/venvs/pls/lib/python3.9/site-packages/pls/models/mixins/stat.py", line 63, in stat_cells
    cells[column] = function(self.stat, *func_args)
  File "/Users/adamparkin/.local/pipx/venvs/pls/lib/python3.9/site-packages/pls/fs/stats.py", line 86, in get_formatted_user
    pw_name = getpwuid(stat.st_uid).pw_name
KeyError: 'getpwuid(): uid not found: 502'
```

## Reproduction

See description

## Expectation

It not to throw an exception and produce the same output as `ls`

## Screenshots

N/A

## Environment
 - Device: Macbook Air (M1)
 - OS: Big Sur 11.6.4
 - Browser: Not applicable?
 - Version: Not applicable?
 - Other info: 

## Additional context

These folders with the missing user got created by copying directories from a backup from another old Macbook to this machine.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by dhruvkb** [comment]
Thanks @pzelnip for reporting this bug, I've just released version 1.13.1 containing commit 6bfdd052bcaffdc6c84d61fa19ba55f284c51753 that should fix this bug. Could you see if that works?

**Comment by pzelnip** [comment]
It does indeed!  Thank you very much for the prompt fix!
